lldap_cli_wrapper.py

```
@@ -0,0 +1,73 @@
+                "Content-Type": "application/json",
+                "Origin": authelia_url,
+                "Referer": f"{authelia_url}/reset-password/step1",
+            },
+            timeout=10,
+        )
+        if response.status_code not in {200, 204}:
+            return (
+                False,
+                f"User created but Authelia reset failed: {response.status_code} - {response.text}",
+            )
+    except Exception as e:  # noqa: BLE001
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)